README.md

```diff
@@ -79,47 +79,74 @@ Each block is 4 KiB large.
 The superblock is the first block of the partition (block 0). It contains the partition's metadata, such as the number of blocks, number of inodes, number of free inodes/blocks, ...
 
 ### Inode store
-Contains all the inodes of the partition. The maximum number of inodes is equal to the number of blocks of the partition. Each inode contains 40 B of data: standard data such as file size and number of used blocks, as well as a simplefs-specific field called `index_block`. This block contains:
+Contains all the inodes of the partition. The maximum number of inodes is equal to the number of blocks of the partition. Each inode contains 40 B of data: standard data such as file size and number of used blocks, as well as a simplefs-specific field called `dir_block/ei_block`. This block contains:
```

Comment:
The combination of `dir_block/ei_block` sounds misleading. Can you distinguish from each other?

Reply:
ok!

Reply:
how about "simplefs-specific union field contain `dir_block` and `ei_block`."
and I modify "Each inode contains 40 B" to "Each inode contains 72 B"